central/metrics/custom/image_vulnerabilities/labels.go

```diff
@@ -0,0 +1,23 @@
+package image_vulnerabilities
+
+import (
+	"github.com/stackrox/rox/central/metrics/custom/tracker"
+	"github.com/stackrox/rox/generated/storage"
+)
+
+var (
+	lazyLabels = []tracker.LazyLabel[finding]{
+		{Label: "Cluster", Getter: func(f *finding) string { return f.deployment.GetClusterName() }},
+	}
+
+	labels = tracker.MakeLabelOrderMap(lazyLabels)
+)
+
+type finding struct {
```

Comment:
Looking at your next PR, I think some variation of the PR description should make it into a comment here:

```
Every piece of source data sent for aggregation is called a finding. The aggregation process builds an aggregation key, which serves as the key for the aggregated numbers map.

For example, the keys for a single "Severity" label will be 4 known severity levels (LOW to CRITICAL), the aggregation map will have a key for each severity, and the number of findings as the value.
For multiple labels, the key is a joined value of all labels.
```